### DISCUSSION TITLE
ImGui cannot run on Linux arm64.

### DISCUSSION THREAD

**Discussion by qian-o** [dotnet/Silk.NET#1763] [Q&A] [answered]
There is no linux-arm64 dependency available in ImGui.NET. If I want to use it in that environment, what do I need to do?

**Comment by Beyley** [comment]
You can build the correct version of cimgui manually for linux-arm64, then put that into the release folder, it should load it

**Reply by qian-o** [reply]
Okay, will Silk.NET be expanded to support more in the future, or where can I find pre-made dependencies?

**Reply by Beyley** [reply, marked as answer]
This is something that should be brought upstream to ImGui.NET, as they are the ones packaging the ImGui related native libraries (they also are missing windows-arm64 libs, which has caused me problems in the past too)

We already try to support arm64 where possible with the native libraries we package (although we are still missing [a lot](https://github.com/dotnet/Silk.NET/issues/727))
linux-arm64 is a place we are *really* lacking right now though, but it is planned that we will support ARM64 Linux on as many of our native libraries as possible

**Reply by qian-o** [reply]
Okay, thank you. If there is an opportunity, I will contribute my small efforts.
